{"level":"error","message":"Não foi possível conectar ao banco de dados: Failed to connect to sqlserver.quadroanuncios.com.br:1433 in 15000ms","name":"SequelizeConnectionError","original":{"code":"ETIMEOUT"},"parent":{"code":"ETIMEOUT"},"stack":"SequelizeConnectionError: Failed to connect to sqlserver.quadroanuncios.com.br:1433 in 15000ms\n    at ConnectionManager.connect (C:\\repos\\Back_ControleGastos\\node_modules\\sequelize\\lib\\dialects\\mssql\\connection-manager.js:119:17)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async ConnectionManager._connect (C:\\repos\\Back_ControleGastos\\node_modules\\sequelize\\lib\\dialects\\abstract\\connection-manager.js:222:24)\n    at async C:\\repos\\Back_ControleGastos\\node_modules\\sequelize\\lib\\dialects\\abstract\\connection-manager.js:174:32\n    at async ConnectionManager.getConnection (C:\\repos\\Back_ControleGastos\\node_modules\\sequelize\\lib\\dialects\\abstract\\connection-manager.js:197:7)\n    at async C:\\repos\\Back_ControleGastos\\node_modules\\sequelize\\lib\\sequelize.js:305:26\n    at async Sequelize.authenticate (C:\\repos\\Back_ControleGastos\\node_modules\\sequelize\\lib\\sequelize.js:457:5)","timestamp":"2025-03-03T15:15:49.850Z"}
